config/locales/en/lookup.yml

```diff
@@ -10,8 +10,11 @@ en:
         input_error: Enter a valid postcode
         error_description: Enter a valid postcode
     lookup:
-      input_label: Enter your postcode
-      hint_text: For example LS1 1UR
+      input_label: Enter a full postcode 
+      hint_text: |
```

Comment:
Has a content designer approved this? It looks confusing to me, it could be read as LS1 1UR being an example of accurate information about the area. If the sentences were the other way around it would work better.

Reply:
I see what you mean, it was content approved but I can go back to them with your point and see what they say.

Reply:
Hi @maxgds! It was a content designer who added the ticket with the mocked up example. The content here matches what they asked for.

Reply:
They have agreed to your suggestion, I will make the change and ask for re-review 🙂